flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -143,6 +143,34 @@ public SentinelSequenceNumber toSentinelSequenceNumber() {
 	/** The interval after which to consider a shard idle for purposes of watermark generation. */
 	public static final String SHARD_IDLE_INTERVAL_MILLIS = "flink.shard.idle.interval";
 
+  /**
+	 * The base backoff time between each describeStream attempt.
+	 * Different tag name to distinguish from "flink.stream.describe.backoff.base"
+	 * since the latter is deprecated.
+	 */
+	public static final String DYNAMODB_STREAMS_DESCRIBE_BACKOFF_BASE =
```

Comment:
Remove these and remove the `@Deprecated` annotations from the prior properties.

Reply:
@tweise  are you ok with removing this function `KinesisProxy::replaceDeprecatedConsumerKeys` as well ?

Reply:
It would be good to modify the method to map/replicate the keys but not remove them (since you need them for dynamo). Add a comment that this is for backward compatibility for the regular proxy that is now using listShards instead of describeShards. We can remove the mapping in later release.

Reply:
OK.